grails-app/services/de/iteratec/osm/measurement/environment/BrowserService.groovy

```diff
@@ -27,33 +28,37 @@ class BrowserService {
         return Collections.unmodifiableSet(result)
     }
 
-    List<Browser> findAllByNameOrAlias(List<String> browserNameOrAlias) {
+    List<Browser> findAllByName(List<String> browserNames) {
         List<Browser> result = []
-        browserNameOrAlias.each {
+        browserNames.each {
```

Comment:
Note that this list may includes `nuil` values which aren't checked in `QueueAndJobStatusService.groovy`